toolchains/symbiflow.py

```diff
@@ -1001,6 +1003,451 @@ def check_env():
         }
 
 
+class NextpnrFPGAInterchange(Toolchain):
```

Comment:
We should have a generic Nextpnr class for all different architectures available, such that we do not duplicate code across archs.

Reply:
PTAL: [NextpnrGeneric](https://github.com/SymbiFlow/fpga-tool-perf/pull/323/files#diff-69ab290f878020ed67b1008338e82df43674e16b1e45733d5f0d8e04b6601141R585-R604)